ldk/go/examples/search-global/Makefile

```diff
@@ -0,0 +1,54 @@
+REPO := github.com/open-olive/loop-development-kit/ldk/go/example/clipboard
```

Comment:
I think this should be:
`REPO := github.com/open-olive/loop-development-kit/ldk/go/example/search-global`

Reply:
Whoops - thanks!

Also, `github.com/open-olive/loop-development-kit/ldk/go/example` should be changed to *examples* everywhere, right?

Reply:
Oh yes. Good catch. It should be plural.

Reply:
Cool - I'll change it everywhere. 👍